wp-content/themes/theme_name/404.php

```diff
@@ -1,6 +1,10 @@
 <?php
+/**
+ * GSS error page
+ */ 
+
 get_header();
 
-esc_html_e( 'ERROR 404', 'text_domain' ):
+echo __( 'ERROR 404', 'text_domain' );
```

Comment:
This should be escaped. In general, we should escape all strings, even ones that are translatable, for security sake.